### PR TITLE
Bugfix - Google Sheets

### LINKED ISSUE #5406: [BUG] Add Single Row jump some columns when a column don't have name
**Describe the bug**
When we don't set an name to an column the action `Add Single Row` will insert the data after this not named column.

**To Reproduce**
1. Create some columns and don't give an name to one, like this:
image

2. Try to add a single row

image

3. See the data inserted before the empty column
image

### LINKED ISSUE #5604: [ACTION] Harvest - Get Projects
**Description**
Querry all the project data from Harvest

**API Docs**
https://api.harvestapp.com/v2/projects

### PR DESCRIPTION
**Pull request by vellames**
Closes #5406

### CODE REVIEW

File: components/google_sheets/actions/add-single-row/add-single-row.mjs [hunk 1]
```diff
@@ -102,6 +102,7 @@ export default {
       convertedIndexes,
     } = this.googleSheets.arrayValuesToString(cells);
 
+    console.log(arr, convertedIndexes);
```
Comment (feyzullah) [inline components/google_sheets/actions/add-single-row/add-single-row.mjs:105]: Do we need this `console.log` here? If we don't can we remove it?

**Review by feyzullah — APPROVED** [review]
Hi @vellames this one looks good to me. Added only one comment. Can you please check?

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Google_Sheet_5406.pdf](https://github.com/PipedreamHQ/pipedream/files/10763001/Google_Sheet_5406.pdf)

**Review by feyzullah — APPROVED** [review]

**Pull request merged**